pkg/client/tables.go

```diff
@@ -343,3 +351,104 @@ func (c *Client) DeleteTableEntry(ctx context.Context, entry *p4_v1.TableEntry)
 
 	return c.WriteUpdate(ctx, update)
 }
+
+func (c *Client) NewActionProfileMemberEntry(
```

Comment:
s/NewActionProfileMemberEntry/NewActionProfileMember

same for the other methods below